build.fsx

```diff
@@ -0,0 +1,302 @@
+open Fake.Core
+open System
+open Fake.DotNet.Testing
+
+#r "paket:
+storage: packages
+nuget Fake.IO.FileSystem
+nuget Fake.DotNet.MSBuild
+nuget Fake.DotNet.Testing.XUnit2
+nuget Fake.DotNet.AssemblyInfoFile 
+nuget Fake.DotNet.NuGet prerelease
+nuget Fake.DotNet.Cli
+nuget Fake.Core.Target
+nuget Fake.DotNet.Cli
+nuget Fake.Api.Github
+nuget xunit.runner.console
+nuget NuGet.CommandLine
+nuget Fake.Core.ReleaseNotes //"
+
+#load "./.fake/build.fsx/intellisense.fsx"
+
+open Fake.Core
+open Fake.IO
+open Fake.IO.FileSystemOperators
+open Fake.IO.Globbing.Operators
+open Fake.DotNet
+open Fake.DotNet.NuGet
+
+// ---------
+// Configuration
+// ---------
+
+let project = "OpenTK"
+
+let authors = [ "Team OpenTK" ]
+
+let summary = "A set of fast, low-level C# bindings for OpenGL, OpenGL ES and OpenAL."
+
+let description = "The Open Toolkit is set of fast, low-level C# bindings for OpenGL, OpenGL ES and OpenAL. It runs on all major platforms and powers hundreds of apps, games and scientific research."
+
+let tags = "OpenTK OpenGL OpenGLES GLES OpenAL C# F# .NET Mono Vector Math Game Graphics Sound"
+
+let copyright = "Copyright (c) 2006 - 2019 Stefanos Apostolopoulos <stapostol@gmail.com> for the Open Toolkit library."
+
+let solutionFile  = "OpenTK.sln"
+
+let gitOwner = "opentk"
+
+let gitHome = "https://github.com/" + gitOwner
+
+// The name of the project on GitHub
+let gitName = "opentk"
+
+// The url for the raw files hosted
+let gitRaw = Environment.environVarOrDefault "gitRaw" "https://raw.github.com/opentk"
+
+// Read additional information from the release notes document
+let release = ReleaseNotes.load "RELEASE_NOTES.md"
+
+
+// ---------
+// Properties
+// ---------
+
+let binDir = "./bin/"
+let buildDir = "./bin/build/"
+let testDir  = "./bin/test/"
+
+// ---------
+// Projects & Assemblies
+// ---------
+
+let toolProjects =
+    !! "src/Generators/**/*.??proj"
+
+let releaseProjects =
+    !! "src/**/*.??proj"
+    -- "src/Generators/**"
+
+let testProjects =
+    !! "tests/**/*.??proj"
+
+let allProjects =
+    [toolProjects ; releaseProjects; testProjects ]
+    |> Seq.concat
+    |> Seq.toList
+
+let testAssemblies = "tests/**/obj/Release/*Tests*.dll"
+
+let nugetCommandRunnerPath = ".fake/build.fsx/packages/NuGet.CommandLine/tools/NuGet.exe" |> Fake.IO.Path.convertWindowsToCurrentPath
```

Comment:
Maybe move the nuget.exe path into a variable on top of the script as well

Reply:
I'll see later if I can remove it, to my memory I think I can remove it...